packages/hdwallet-native/src/crypto/isolation/adapters/bip32.ts

```diff
@@ -47,7 +48,7 @@ export class BIP32Adapter extends ECPairAdapter implements BIP32InterfaceAsync {
     networkOrParent?: BIP32Adapter | Network,
     index?: number
   ): Promise<BIP32Adapter> {
-    await btccryptoReady;
+    await Promise.all([btccryptoReady, ECPairAdapter.prepare()]);
```

Comment:
How is this change related to lazy loading?

Can you add a comment explaining _why_ `prepare()` needs to be called here?

Reply:
Extra comments added; also see https://github.com/shapeshift/hdwallet/pull/516#discussion_r852466460.